static/app/components/replays/breadcrumbs/utils.tsx

```diff
@@ -32,11 +38,18 @@ export function getDescription(crumb: Crumb) {
     }
   }
 
+  if (crumb.category === 'replay.mutations') {
+    return t(
+      'A large number of mutations was detected (%s). This can slow down the Replay SDK and impact your customers.',
+      crumbData?.count
+    );
```

Comment:
This is a large number happening in a short timeframe?

Reply:
Yeah

Reply:
Technically it's in the same tick (I believe - hard to say how the mutationobserver internals work in detail), so _very short_ timespan.